Dockerfile

```diff
@@ -1,12 +1,16 @@
 # Start with a base image containing Java runtime
-#FROM openjdk:11.0.6-stretch
-#FROM openjdk:12-jdk-alpine
-FROM openjdk:12-jdk-oraclelinux7
+# FROM openjdk:11.0.6-stretch
+# FROM openjdk:12-jdk-alpine
+# FROM openjdk:12-jdk-oraclelinux7
+FROM openjdk:11.0.12-jre
+
 # Add Maintainer Info
 LABEL maintainer="gabriele.deluca@eng.it"
 
-#Install whois service
-RUN yum install -y whois
+# Install whois service
+# RUN yum install -y whois
+# RUN set -eux; 	apt-get update; 	apt-get install -y --no-install-recommends
```

Comment:
Is this commented line correct if uncommented?
Please add comment what we do with line below (installing whois)

Reply:
Line 12 was used in test phase. It is not necessary here.